static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
Sorry - the array syntax would not be detected as a static require though. Also the use of a variable `bootUrl` is not statically detectable anyway.

Array requires definitely pipe through directly to `System.import`. Perhaps try replacing that call with a direct `System.import(bootUrl).then(function(interactive) {})` and see if that works?